src/tracks/ui/TrackView.h

```diff
@@ -13,6 +13,7 @@ Paul Licameli split from class Track
 
 #include <memory>
 #include "CommonTrackPanelCell.h" // to inherit
+#include "XMLAttributeValueView.h"
```

Comment:
Good to see this change.  There are other new inclusions of XMLTagHandler.h that also only need XMLAttributeValueView.h in other files.  Fix those too if you feel like it.  But it's not very important.